code/src/java/pcgen/gui2/facade/Gui2InfoFactory.java

```diff
@@ -859,15 +859,15 @@ private HtmlInfoBuilder getEquipmentHtmlInfo(Equipment equip)
 
 		Integer a = EqToken.getMaxDexTokenInt(pc, equip);
 
-		if (a.intValue() != Constants.MAX_MAXDEX)
```

Comment:
They do not unconditionally error. I'm confused about maintaining the .intValue() helps anything though? It has identical semantics.